### ISSUE TITLE
Low accuracy of kitti model after training complete

### ISSUE THREAD

**Issue by DJmimi35** [NVIDIA/caffe#272]
Hi all,

I have tried to create a model of kitti dataset, and now it works without any error. However, the accuracy stays in low level from about 10 epochs till finish. And the model just can't recognize any car, even the photo from kitti-data/val directory. I've verified with different GPU model(P40 & P100), different caffe version(0.15.1 and 0.15.14), and different numbers of GPU during creating. But the result is very similar(precision(val) accuracy rate around 5% ~ 20%). There might be something I missed in my training steps. The detail info of this system is as below:

GPU is Tesla P40 * 2
CUDA: 8.0.44-1
NVIDIA Driver: 367.48
DIGITS version: 5.1-dev
caffe version: NVcaffe-0.15.14
cuDNN: 5.1.5
nccl: 1.2.3-1

[caffe-config.txt](https://github.com/NVIDIA/caffe/files/602806/caffe-config.txt)
[nvidia-smi.txt](https://github.com/NVIDIA/caffe/files/602807/nvidia-smi.txt)
[nvidia-smi-q.txt](https://github.com/NVIDIA/caffe/files/602808/nvidia-smi-q.txt)

![kitti-low-accuracy](https://cloud.githubusercontent.com/assets/23118096/20471152/54d5d7da-afea-11e6-95d2-2715b0a95627.png)
![kitti-model](https://cloud.githubusercontent.com/assets/23118096/20471155/59589860-afea-11e6-91d2-2586b27a3759.png)

Thanks

**Comment by DJmimi35** [comment]
After adding pretrained model from http://dl.caffe.berkeleyvision.org/bvlc_googlenet.caffemodel , the mAP rises to about 64 which is closed to the value in NVIDIA DIGITS example, and it could recognize the most of car box from kitti-data/val/images. So I decide to close this issue, thanks.
![accuracy-rise](https://cloud.githubusercontent.com/assets/23118096/20658825/4b119106-b505-11e6-91f8-9cf03433065b.png)
